sdk/src/DocuSign.eSign/Client/ApiClient.cs

```diff
@@ -409,7 +408,7 @@ public object Deserialize(IRestResponse response, Type type)
         /// <param name="type">Object type.</param>
         /// <param name="headers"></param>
         /// <returns>Object representation of the JSON string.</returns>
-        public object Deserialize(byte[] content, Type type, IList<Parameter> headers = null)
+        public object Deserialize(byte[] content, Type type)
```

Comment:
@RajRele this is a public method. Hopefully no one is using it otherwise their code might need to be updated

Reply:
`Parameter` class is obsolete, we want to remove the usage and comply to a newer usage. Even if this is a public method the `headers` object is unused in this method, users should not be affected :)